tests/test_snmp_notification_receiver.rb

```diff
@@ -59,16 +59,47 @@ def test_create_destroy_single
                                           source_interface: 'ethernet1/3')
 
     assert_includes(Cisco::SnmpNotificationReceiver.receivers, id)
-    assert_equal(Cisco::SnmpNotificationReceiver.receivers[id], receiver)
+    assert_equal(receiver, Cisco::SnmpNotificationReceiver.receivers[id])
 
-    assert_equal(Cisco::SnmpNotificationReceiver.receivers[id].source_interface,
-                 'ethernet1/3')
-    assert_equal(Cisco::SnmpNotificationReceiver.receivers[id].port, '45')
-    assert_equal(Cisco::SnmpNotificationReceiver.receivers[id].type, 'informs')
-    assert_equal(Cisco::SnmpNotificationReceiver.receivers[id].username, 'ab')
-    assert_equal(Cisco::SnmpNotificationReceiver.receivers[id].version, '3')
-    assert_equal(Cisco::SnmpNotificationReceiver.receivers[id].vrf, 'red')
-    assert_equal(Cisco::SnmpNotificationReceiver.receivers[id].security, 'priv')
+    assert_equal('ethernet1/3',
+                 Cisco::SnmpNotificationReceiver.receivers[id].source_interface)
+    assert_equal('45', Cisco::SnmpNotificationReceiver.receivers[id].port)
+    assert_equal('informs', Cisco::SnmpNotificationReceiver.receivers[id].type)
+    assert_equal('ab', Cisco::SnmpNotificationReceiver.receivers[id].username)
+    assert_equal('3', Cisco::SnmpNotificationReceiver.receivers[id].version)
+    assert_equal('red', Cisco::SnmpNotificationReceiver.receivers[id].vrf)
+    assert_equal('priv', Cisco::SnmpNotificationReceiver.receivers[id].security)
+
+    receiver.destroy
+    refute_includes(Cisco::SnmpNotificationReceiver.receivers, id)
+  end
+
+  def test_create_destroy_single_ipv6
+    id = '2004::7'
+    refute_includes(Cisco::SnmpNotificationReceiver.receivers, id)
+
+    receiver = \
+      Cisco::SnmpNotificationReceiver.new(id,
+                                          instantiate:      true,
+                                          type:             'informs',
+                                          version:          '3',
+                                          security:         'priv',
+                                          username:         'ab',
+                                          port:             '45',
+                                          vrf:              'red',
+                                          source_interface: 'ethernet1/3')
```

Comment:
For better portability between platforms, use `interfaces[0]` and `interfaces[1]` (API from ciscotest.rb) instead of hard-coding the interface names.